src/lib.rs

```diff
@@ -429,3 +429,22 @@ where
         Self::from_slice(slice)
     }
 }
+
+impl<T, const N: usize> From<[T; N]> for StaticVec<T, N> {
+    fn from(array: [T; N]) -> Self {
+        let mut vec = Self::new();
+        for element in array {
+            unsafe { vec.push_unchecked(element) };
+        }
+        vec
+    }
+}
+
+impl<T, const N: usize> From<StaticVec<T, N>> for [T; N] {
+    fn from(mut vec: StaticVec<T, N>) -> Self {
+        unsafe {
+            vec.len = 0;
+            ptr::read(vec.as_ptr() as *const [T; N])
```

Comment:
What about the situation when `vec.len < N`?
In such case uninitialized memory will be assumed initialized which is [undefined behavior](https://doc.rust-lang.org/std/mem/union.MaybeUninit.html#initialization-invariant).

Maybe it's better to implement [`TryFrom`](https://doc.rust-lang.org/std/convert/trait.TryFrom.html) instead of `From`?

Reply:
Ah yes, I didn't think of that case.

Reply:
Ah yes, I didn't think of that case.